tests/decorators/decorators.go

```diff
@@ -17,15 +17,16 @@ var (
 	SigMonitoring        = Label("sig-monitoring")
 
 	// HW
-	GPU         = Label("GPU")
-	VGPU        = Label("VGPU")
-	SEV         = Label("SEV")
-	SRIOV       = Label("SRIOV")
-	StorageReq  = Label("storage-req")
-	Multus      = Label("Multus")
-	Macvtap     = Label("Macvtap")
-	Invtsc      = Label("Invtsc")
-	KSMRequired = Label("KSM-required")
+	GPU                  = Label("GPU")
+	VGPU                 = Label("VGPU")
+	SEV                  = Label("SEV")
+	SRIOV                = Label("SRIOV")
+	StorageReq           = Label("storage-req")
+	BlockStorageRequired = Label("block-storage-required")
```

Comment:
I added decorators for this the other day - https://github.com/kubevirt/kubevirt/pull/13137

Reply:
thanks @akalenyu for being one step in front of me :) will close this PR then